src/main/java/com/accolite/pru/health/AuthApp/security/JwtTokenProvider.java

```diff
@@ -40,11 +45,17 @@ public JwtTokenProvider(@Value("${app.jwt.secret}") String jwtSecret, @Value("${
      */
     public String generateToken(CustomUserDetails customUserDetails) {
         Instant expiryDate = Instant.now().plusMillis(jwtExpirationInMs);
+        String authorities = customUserDetails
+                .getAuthorities()
+                .stream()
+                .map(GrantedAuthority::getAuthority)
+                .collect(Collectors.joining(","));
         return Jwts.builder()
                 .setSubject(Long.toString(customUserDetails.getId()))
                 .setIssuedAt(Date.from(Instant.now()))
                 .setExpiration(Date.from(expiryDate))
                 .signWith(SignatureAlgorithm.HS512, jwtSecret)
+                .claim("authorities", authorities)
```

Comment:
Instead of hardcoding, can you please define a `private static final String` so that it is effectively a constant

Reply:
Will do it ASAP